src/lib/isBIC.js

```diff
@@ -1,6 +1,7 @@
 import assertString from './util/assertString';
 
-const isBICReg = /^[A-z]{4}[A-z]{2}\w{2}(\w{3})?$/;
+// https://en.wikipedia.org/wiki/ISO_9362
+const isBICReg = /^[A-Z]{6}[A-Z0-9]{2}([A-Z0-9]{3})?$/;
```

Comment:
```suggestion
const isBICReg = /^[A-Za-z]{6}[A-Za-z0-9]{2}([A-Za-z0-9]{3})?$/;
```

Reply:
it looks like the spec is uppercase only (i'm inferring that as the standard itself costs 38 CHF to buy).

but i guess this keeps it working closer to how it did before, just pushed the change.

Reply:
Yes, if we disallow lowercase it would become a breaking change.

Reply:
nice catch @fedeci we should probably keep it as it is until the next major version